src/client/src/apps/StyleguideRoute/components/FormGrid.tsx

```diff
@@ -55,36 +56,22 @@ const FormGrid = () => {
   return (
     <Grid>
       <div></div>
-      <div>Text</div>
-      <div>Figures</div>
-
-      <div>Normal</div>
-      <Input type="text" placeholder="Prompt" label="label" />
-      <Input type="number" placeholder="Prompt" label="label" />
-
-      <div>Hover</div>
-      <HoveredInput type="text" placeholder="Prompt" label="label" />
-      <HoveredInput type="number" placeholder="Prompt" label="label" />
-
-      <div>Active</div>
-      <ActiveInput type="text" placeholder="Prompt" label="label" />
-      <ActiveNumberInput type="number" placeholder="Prompt" label="label" />
-
-      <div>Disabled</div>
-      <Input type="text" placeholder="Prompt" disabled label="label" />
-      <Input type="number" placeholder="Prompt" disabled label="label" />
-
-      <div>Populated</div>
-      <Input type="text" placeholder="Prompt" value="Text" label="label" />
-      <Input type="number" placeholder="Prompt" value="1000" label="label" />
+      <div></div>
+      <div></div>
+      <div></div>
+      <div></div>
```

Comment:
Why the empty divs?

Reply:
needed an empty line but fixed with a margin